src/v/datalake/tests/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+find_package(Arrow REQUIRED)
+find_package(Parquet REQUIRED)
```

Comment:
add to cmake/dependencies.cmake to support open source build. the deps existed on fedora so no changes are needed, not sure about ubuntu.